client/my-sites/email/google-sale-banner/index.tsx

```diff
@@ -72,6 +72,7 @@ const GoogleSaleBanner: FunctionComponent< GoogleSaleBannerProps > = ( { domains
 
 	// Verify that we have a percentage discount
 	if (
+		! googleWorkspaceProduct ||
 		! hasDiscount( googleWorkspaceProduct ) ||
 		( ! googleWorkspaceProduct?.sale_coupon?.discount ?? null )
```

Comment:
I don't see why we need to change that condition since `hasDiscount()` will check for the existence of the product anyway - did I miss something 🙂?

Reply:
The types for `hasDiscount()` require an `object` as the parameter. I suppose we could change the types there to support falsy values instead of this.

Reply:
I'll make that change and add it to https://github.com/Automattic/wp-calypso/pull/59649